projects/aca-content/src/lib/ui/overrides/ay11.scss

```diff
@@ -157,13 +157,21 @@
     }
   }
 
+  .mat-datetimepicker-calendar-body-active .mat-datetimepicker-calendar-body-cell-content {
+    outline: 2px solid var(--theme-blue-button-color);
```

Comment:
please never embed color names into the variables of the theme! theming is exactly for changing the colors, so people will have to set "red" to a variable that says "blue"

Reply:
This variable and CSS is something that already existed in the codebase, and is showing up as a change here because we've changed the CSS rule slightly. This variable is also being used very widely in the application. We can change the variable name as requested, but that might impact a lot of places, which may or may not be related to the linked issue, i.e. datetimepicker. 

Some of the areas where I saw usages of this variable include - 
- CSS rules for adf-datatable selected item
- Mat Slide toggle
- View-Profile component
- Open-In-App component

Reply:
okay, but we should not hide the problem just because it was there before you came... to solve this, you can introduce a proper CSS variable and use it here? I see that it was introduced here: https://github.com/Alfresco/alfresco-content-app/blob/develop/projects/aca-content/src/lib/ui/variables/variables.scss#L68. Just add a proper variable, and reference that in datetime picker.

Reply:
Added new variables for border color. New variable name is `$datetimepicker-cell-focus-border-color` & `--theme-datetimepicker-cell-focus-border`